### PR TITLE
[Release/Demo] Indiana Jones 1

### PR DESCRIPTION
**Pull request by j-brammer**
Deploy PR for Indiana Jones 1

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -49,6 +49,8 @@ contentful:
     query: fields.category=common
     map:
       content_block: content
+  page:
+    query: "fields.collections[exists]=false"
```
Comment (tcmacdonald) [inline _config.yml]: FYI @j-brammer This update relies on migrations introduced via crdschurch/crds-contentful-migrations#157

File: cypress/integration/liveStream/live_stream_player_spec.js [hunk 2]
```diff
@@ -9,9 +9,10 @@ function hideRollCall() {
 }
 
 function getYoutubeId(youtubeURL) {
-  const regex = '\\w+://\\S+/(?:watch\\?v=)?(\\w{11}).*';
-  const match = new RegExp(regex).exec(youtubeURL);
-  return match[1];
+  const regex = /youtu(?:be|.be)?(?:.+)\/(?:.+v=)?(.{11})/;
```
Comment (tcmacdonald) [inline cypress/integration/liveStream/live_stream_player_spec.js]: I might steal this regex for crdschurch/crds-serverless

File: index.html [hunk 3]
```diff
@@ -2,10 +2,10 @@
 layout: default
 monetate_page_type: index-main
 meta:
-  title: Crossroads
-  description: No matter what your beliefs, you are welcome here.
-  image:
-    url: "//crds-cms-uploads.imgix.net/content/images/cr-social-sharing-still-bg.jpg?w=1200&h=630&fit=crop"
```
Comment (tcmacdonald) [inline index.html]: Could removing the indentions on this file cause issues?

File: redirects.csv [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/,/h,200! Role=user
```
Comment (tcmacdonald) [inline redirects.csv]: Creative.

File: _config.yml (hunk 1 above)
Comment (j-brammer) [reply _config.yml]: Created demo PR for contentful-migrations 
https://github.com/crdschurch/crds-contentful-migrations/pull/158

File: index.html (hunk 3 above)
Comment (j-brammer) [reply index.html]: @tcmacdonald Should be fixed now! Great catch.

**Pull request merged**